README.md

```diff
@@ -8,6 +8,15 @@ MMB is the set of Dockerfiles and assets for building Docker images with differe
 
 MMB stands for "Mr. Meeseeks Box". The project name was inspired by the episode ["Meeseeks and Destroy"](https://en.wikipedia.org/wiki/Meeseeks_and_Destroy) of an animated sitcom [Rick and Morty](https://en.wikipedia.org/wiki/Rick_and_Morty).
 
+## References
+
+MMB was mentioned in several articles.
+
+* [Deluge и Transmission на Raspberry Pi](https://pikabu.ru/story/deluge_i_transmission_na_raspberry_pi_5618108) published on [Pikabu](https://pikabu.ru) in Russian.
+* [Nextcloud: не только хранилище](https://cusdeb.com/ru/blog/2018/08/16/nextcloud-%d0%bd%d0%b5-%d1%82%d0%be%d0%bb%d1%8c%d0%ba%d0%be-%d1%85%d1%80%d0%b0%d0%bd%d0%b8%d0%bb%d0%b8%d1%89%d0%b5/) originally published in Linux Foramt [233](http://linuxformat.ru/anons233.phtml) in Russian.
```

Comment:
You made a typo: `Foramt`. On [this](https://github.com/tolstoyevsky/mmb/pull/190/files#diff-04c6e90faac2675aa89e2176d2eec7d8R18) line too.